app/models/company.rb

```diff
@@ -0,0 +1,18 @@
+class Company < ActiveRecord::Base
+  has_many :items
+
+  def self.company_save
+    all_companies = FHServices.new.companies_hash[:companies]
+    all_companies.each do |company|
+      unless Company.find_by(shortname: company[:shortname])
+        Company.create(name: company[:name], shortname: company[:shortname])
```

Comment:
Could `ruby find_or_create_by(name: company[:name], shortname: company[:shortname])` do this for us? Why only `.find_by` `shortname`, but then `.create` with both `name` and `shortname` attributes?